### ISSUE TITLE
Extra space added before single-line nested SCSS maps

### ISSUE THREAD

**Issue by stormwarning** [prettier/prettier#4057]
**Prettier 1.10.2**
[Playground link](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEASAZgGwIYyQAgAoAdKffVAN2wCcBaARgOswFc4AaU8q2ugJmbY2nbhWr0AzEJGkAlAG5SpVFDgBneABMCJMkV70GcrvsKGBJseYmMO4vv3sXJz2wBY3fAKxf6vikxoAHM4egkAS2wAI0w4OihsAFt49Ah6OAAPZIAHOKszFwLyGz5PB385eQUQDhAIHJgI6HVkUFoaCAB3AAVaBFaUbEoICK1akGxNZHRhdU4QaJpsMABrOBgAZRyViKhg5BgadjqtCDAZuYW9+ZoYHuXgpOxLzHm6gCt1TIAhZbWNptknAADJ7OCvd71VgwHIw-iQhY7Gi3ZAgdRgdStOo5Gh7GAAdTGMAAFsgABwABhxnXmBOWOTRuI0YUoELqNDgAEdWBFOQ9sE8XkhZm8FvMkhFDsdxXtgnEAIqsCDwRF1GAxIlaUnIJwgI7YCKYOUAYQgSWeaKg0HZIFY8wAKjFBqL5gBfN1AA)
```sh
--parser scss
```

**Input:**
```scss
$flat: (
  $var-1: value,
  $var-2: value,
  $var-3: value
);

$nested: (
  ($var-1),
  ($var-2),
  ($var-1, $var-2, $var-3, $var-4, $var-5, $var-5, $longer-variable-name-for-example),
  ($var-3),
  ($var-4, $var-5)
);
```

**Output:**
```scss
$flat: (
  $var-1: value,
  $var-2: value,
  $var-3: value
);

$nested: (
   ($var-1),
   ($var-2),

  (
    $var-1,
    $var-2,
    $var-3,
    $var-4,
    $var-5,
    $var-5,
    $longer-variable-name-for-example
  ),
   ($var-3),
   ($var-4, $var-5)
);

```

**Expected behavior:**
- No extra whitespace added before parens
- No extra line before a reformatted long line

```scss
$nested: (
  ($var-1),
  ($var-2),
  (
    $var-1,
    $var-2,
    $var-3,
    $var-4,
    $var-5,
    $var-5,
    $longer-variable-name-for-example
  ),
  ($var-3),
  ($var-4, $var-5)
);
```

**Comment by stormwarning** [comment]
Happy to PR a fix, but I couldn't quite figure out where this formatting is coming from 🤷‍♂️

**Comment by stormwarning** [comment]
Looks like both issues are fixed in 1.11 thanks to https://github.com/prettier/prettier/pull/3930 and https://github.com/prettier/prettier/pull/3814 👌
